lib/Config.js

```diff
@@ -737,6 +737,11 @@ class Config extends EventEmitter {
     getLocationConstraintType(locationConstraint) {
         return this.locationConstraints[locationConstraint].type;
     }
+
+    setRestEndpoints(restEndpoints) {
+        this.restEndpoints = restEndpoints;
```

Comment:
Please validate that it is the correct format first and check that each endpoint value is a known location.

Reply:
Updated, added the "known location part" that did not exist for the initial config load :)

Reply:
I was hoping you would :)